apps/dashboard/pages/[endpoint].tsx

```diff
@@ -236,6 +288,110 @@ const ModelPage: FC<ModelPageProps> = ({ endpoint }) => {
         >
           {endpoint === 'posts' && selectedModel && (
             <Box p={4} rounded="md" bg="white" shadow="md">
+              {/* todo: button to generate posts using AI */}
+              <Heading p={4} color='black'>Post Generator</Heading>
+              <Box p={4}>
+                <form onSubmit={handleSubmit}>
+                  <FormLabel>Content</FormLabel>
+                  <Textarea
+                    name='prompt'
+                    placeholder='Enter a content...'
+                    value={input}
+                    onChange={handleInputChange}
+                    mb={4}
+                    size='xs'
+                    required
+                  />
+                  <Flex gap={10} mb={4} >
+                    <Box>
+                      <FormLabel>Number of Posts</FormLabel>
+                      <NumberInput
+                        step={1}
+                        min={0}
+                        max={40}
+                        defaultValue={5}
+                        onChange={(a, b) => setNumberOfPosts(b)}
+                      >
+                        <NumberInputField />
+                        <NumberInputStepper>
+                          <NumberIncrementStepper />
+                          <NumberDecrementStepper />
+                        </NumberInputStepper>
+                      </NumberInput>
+                    </Box>
+                    <Box>
+                      <FormLabel>Character Limit</FormLabel>
+                      <NumberInput
+                        step={10}
+                        min={10}
+                        max={200}
+                        defaultValue={150}
+                        onChange={(a, b) => setCharLimit(b)}
+                      >
+                        <NumberInputField />
+                        <NumberInputStepper>
+                          <NumberIncrementStepper />
+                          <NumberDecrementStepper />
+                        </NumberInputStepper>
+                      </NumberInput>
+                    </Box>
+                    <Box flex='1'>
+                      <FormLabel>Language</FormLabel>
+                      <Select
+                        value={language ?? ''}
+                        onChange={e => setLanguage(e.target.value)}
+                      >
+                        {LANGUAGE_OPTIONS.map((opt, idx) => {
+                          return (
+                            <option key={idx}>{opt}</option>
+                          )
+                        })}
+                      </Select>
+                    </Box>
+                  </Flex>
+
+                  <Button
+                    disabled={isLoading}
+                    type='submit'
+                  >
+                    Generate
```

Comment:
You can add translations to all `common.json` files in each project.

```suggestion
                   {t('generate')}
```

Reply:
The simplest way to add a new translation to the common.json files can be to use VSCode's global search and replace tool.

Search

```jsonc
"give-feedback": "Give feedback",
```

```jsonc
// "give-feedback": "Feedback geven",
// "give-feedback": "Geri Bildirimde Bulun",
```

Replace

```json
"generate": "Generate",
"give-feedback": "Give feedback",
```